_posts/2021-06-22-solidity-0.8.6-release-announcement.md

```diff
@@ -39,9 +40,9 @@ The peephole optimizer and the jumpdest remover are activated even when
 setting ``settings.optimizer.enabled`` to ``false`` in Standard JSON
 and have to be explicitly disabled in the detailed settings.
 
-The bug was that omitting ``enabled`` was not equivalent to setting it to
-``false``: Omitting all optimizer settings was equivalent to explicitly
-disabling all optimizer components.
+The bug was that providing a value for ``settings.optimizer`` but not including
+``enabled`` in it was not equivalent to setting ``enabled`` to ``false``.
+It was equivalent to explicitly disabling all optimizer components.
 
```

Comment:
If `settings.optimizer` was not provided at all, the bug did not manifest.

Reply:
Added.